src/LibraryViewExtensionWebView2/web/library/build/librarie.min.js.LICENSE.txt

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
hmm this file needs to be hard copied over?

Reply:
I included that file as did in NotificationsCenter, do you want me to remove it?